package.json

```diff
@@ -29,7 +30,8 @@
     "proto3-json-serializer": "^1.0.0",
     "protobufjs": "7.2.3",
     "protobufjs-cli": "1.1.1",
-    "retry-request": "^5.0.0"
+    "retry-request": "^5.0.0",
+    "tar": "^6.1.13"
```

Comment:
This needs to be reverted, we don't want to have `tar` runtime dependency in GAX